assets/app/scripts/controllers/build.js

```diff
@@ -113,7 +113,7 @@ angular.module('openshiftConsole')
 
       (function initLogs() {
         angular.extend($scope, {
-          logs: [],
+          logs: [{text: 'Logs for '+ $routeParams.build}, {text: ' '}],
```

Comment:
Unsure about making this part of the log text. I think it might be better beside the status above the log. Did you choose this so it displays in the chromeless view?